requirements/edx/github.txt

```diff
@@ -29,10 +29,10 @@ git+https://github.com/mitocw/django-cas.git@60a5b8e5a62e63e0d5d224a87f0b489201a
 -e git+https://github.com/edx/bok-choy.git@82d2f4b72e807b10d112179c0a4abd810a001b82#egg=bok_choy
 -e git+https://github.com/edx-solutions/django-splash.git@7579d052afcf474ece1239153cffe1c89935bc4f#egg=django-splash
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@release-2015-04-07T13.05#egg=edx-ora2
+-e git+https://github.com/singingwolfboy/edx-ora2.git@update-pytz#egg=edx-ora2
```

Comment:
I had to do that in order to get the automated tests to pass. Because `edx-platform` and `edx-ora2` share the same virtualenv, the dependency versions must be kept in sync with each other. (Same deal with `ease`.) Once the edx fork of `edx-ora2` has the same version of `pytz` as the edx fork of `edx-platform`, then we can make them depend on each other again. Does that make sense?